### DISCUSSION TITLE
XML encoding

### DISCUSSION THREAD

**Discussion by ismailatitallah** [OP-TED/eForms-SDK#738] [Q&A] [answered]
Bonjour, 
Je soumets des avis via l'api de ENotice2. Dans la platefome ENotices2, le xml téléchargé a des problèmes d'encoding.
Voici le xml soumis 
[notice-5104353-4-dc8d9eba-8c18-49e1-aeaf-126acc6ba169.zip](https://github.com/OP-TED/eForms-SDK/files/13121931/notice-5104353-4-dc8d9eba-8c18-49e1-aeaf-126acc6ba169.zip)

**Comment by YvesJo** [comment, marked as answer]
Bonjour,
Pour les questions non relatives au SDK, merci de vous adresser directement au support ted@publications.europa.eu
